web.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM 319273870782.dkr.ecr.us-east-1.amazonaws.com/covid-risk-score:shiny-verse as base
+FROM sha256:8d17ac6ce0f0d7fd6f6d4254425b5bec4e538b9bcc0df66a93985d4f31cca01a as base 

```

Comment:
@shruthirames this looks temporary, can you leave a comment on how to update this?